matRad/optimization/projections/matRad_BEDProjection.m

```diff
@@ -0,0 +1,39 @@
+classdef matRad_BEDProjection < matRad_EffectProjection
+% matRad_BEDProjection class for BED-based optimization
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2019 the matRad development team. 
+% 
+% This file is part of the matRad project. It is subject to the license 
+% terms in the LICENSE file found in the top-level directory of this 
+% distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part 
+% of the matRad project, including this file, may be copied, modified, 
+% propagated, or distributed except according to the terms contained in the 
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%    
+
+    methods
+        function obj = matRad_BEDProjection()
+        end
+    end
+
+    methods
+        function BED = computeSingleScenario(obj,dij,scen,w)
+                effect = computeSingleScenario@matRad_EffectProjection(obj,dij,scen,w);
+                BED = zeros(dij.doseGrid.numOfVoxels,1);
+                ix = ~(dij.ax{scen}==0);
```

Comment:
Please compare this to the variable RBE weighted dose projection class, because:
- you can not just take `scen` here as currently ax and bx do only vary in ct scenarios, so it needs to be the ct scenario that you query here. It should be correctly done for rbe weighted.
- There should already be indices store (ixDose ?) in the dij object to avoid doing the lookup everytime